frontend/src/v4/constants/issues.ts

```diff
@@ -44,6 +45,11 @@ export const ISSUE_TABS = {
 	ATTACHMENTS: 'Attachments',
 };
 
+export const V5_ISSUE_TABS = {
+	...ISSUE_TABS,
+	SEQUENCING: '4D',
```

Comment:
Even if their content is the same, I think It would not make logical sense. 
![image](https://user-images.githubusercontent.com/43547703/165969143-a369877c-489f-4023-b5d0-4d8689c1d2c2.png)
Take `ISSUE_SEQUENCING_TAB` as a example, it is lowercase, hence not re-used in `ISSUE_TABS`.

Reply:
@carmenfan Yes, most of this code was already there. I can refactor it if you want me to, but I believe at some we will have to rewrite the code for the viewer from scratch anyways for V5

Reply:
nah v4 code is not worth cleaning up since we're ditching it.

Just bear in mind when you write new code